dists/make-release.sh

```diff
@@ -2,18 +2,15 @@
 
 [ "$DEBUG" = "1" ] && set -x
 set -e
+set -x
 
 VERSION=""
-BUILD_ONLY=false
```

Comment:
The mode with `BUILD_ONLY` equals to `false` has never been used. Moreover I want we move away from this script (https://github.com/Scalingo/cli/issues/739) so it's not really an issue to delete a bit of code here.